packages/cli/package.json

```diff
@@ -16,16 +16,19 @@
     "url": "https://github.com/LionWeb-io/lionweb-typescript/issues"
   },
   "dependencies": {
-    "@lionweb/core": "0.6.10-beta.0",
-    "@lionweb/utilities": "0.6.10-beta.0",
+    "@lionweb/core": "0.6.10-beta.2",
+    "@lionweb/utilities": "0.6.10-beta.2",
     "@lionweb/validation": "0.6.3"
   },
   "scripts": {
     "clean": "rm -rf dist/ lionweb-cli-*.tgz",
     "build": "tsc",
     "lint": "eslint src",
     "test": "node dist/lionweb-cli.js",
+    "pre-release-either": "npm run clean && npm install && npm run build",
```

Comment:
what pre-release either means in this case?

Reply:
It's the pre-command for either the `release` or `release-beta` scripts.